dirsrvtests/tests/tickets/ticket2058_test.py

```diff
@@ -0,0 +1,131 @@
+import logging
+import pytest
+import os
+import subprocess
+import time
+import ldap
+from lib389._constants import *
+from lib389.topologies import topology_i2 as topo
+
+DEBUGGING = os.getenv("DEBUGGING", default=False)
+if DEBUGGING:
+    logging.getLogger(__name__).setLevel(logging.DEBUG)
+else:
+    logging.getLogger(__name__).setLevel(logging.INFO)
```

Comment:
You can skip this part. It's in lib389 now :)

Reply:
It was also created by create_test template, 
 Anyway fixing mreynolds389's remark about avoiding to create new testcase in ticket directory will take care of that ...

Reply:
Mark recently did a change to the template create that removes this too :)